cobbler/tftpgen.py

```diff
@@ -229,18 +233,22 @@ def write_all_system_files(self, system, menu_items):
             return
 
         # generate one record for each described NIC ..
-        for (name, interface) in list(system.interfaces.items()):
+        for (name, _) in list(system.interfaces.items()):
```

Comment:
Is the underscore correct?

Reply:
Yes it is: https://stackoverflow.com/a/5893946/4730773

Reply:
In this case the third usecase is the one applicable. I am not using this variable and thus need a throw away operator, which is `_` in Python.

Reply:
Thanks for the clarification.